scripts/uberenv/packages/mfem/package.py

```diff
@@ -369,21 +365,27 @@ def find_optional_library(name, prefix):
         cxxflags = spec.compiler_flags['cxxflags']
 
         if cxxflags:
+            # Add opt/debug flags if they are not present in global cxx flags
```

Comment:
All changes except the added hdf5 lines are just updating the package from the spack repo.